diesel_derives/src/associations.rs

```diff
@@ -35,26 +35,48 @@ fn derive_belongs_to(
         parent_struct,
         foreign_key,
     } = AssociationOptions::from_meta(meta)?;
-    let (impl_generics, ty_generics, where_clause) = generics.split_for_impl();
+    let (_, ty_generics, _) = generics.split_for_impl();
 
     let foreign_key_field = model.find_column(&foreign_key)?;
     let struct_name = &model.name;
     let foreign_key_access = foreign_key_field.name.access();
     let foreign_key_ty = inner_of_option_ty(&foreign_key_field.ty);
     let table_name = model.table_name();
 
+    let mut generics = generics.clone();
+
+    // TODO: Remove this specialcasing as soon as we bump our minimal supported
+    // rust version to >= 1.30.0
+    // Then we could use the `&Option<T> -> `Option<&T>` from implementation in
```

Comment:
Can this comment more explicitly say "Rust 1.30 adds `impl<'a, T> From<&'a Option<T>> for Option<&'a T>`"?